doc/releases/changelog-0.28.0.md

```diff
@@ -651,79 +486,100 @@
 
-* Devices can now disregard observable grouping indices in Hamiltonians through the optional `use_grouping` attribute.
-  [(#3456)](https://github.com/PennyLaneAI/pennylane/pull/3456)
+* The `qcut.py` file in `pennylane/transforms/` has been reorganized into multiple files that are now in `pennylane/transforms/qcut/`.
+  [(#3413)](https://github.com/PennyLaneAI/pennylane/pull/3413)
 
 * A warning now appears when creating a `Tensor` object with overlapping wires, informing that this can lead to undefined behaviour.
   [(#3459)](https://github.com/PennyLaneAI/pennylane/pull/3459)
 
 * Extended the `qml.equal` function to `qml.ops.op_math.Controlled` and `qml.ops.op_math.ControlledOp` objects.
   [(#3463)](https://github.com/PennyLaneAI/pennylane/pull/3463)
 
-* Extended the `qml.equal` function to `Pow`, `SProd`, `Exp` and `Adjoint` objects.
+* Nearly every instance of `with QuantumTape()` has been replaced with `QuantumScript` construction.
+  [(#3454)](https://github.com/PennyLaneAI/pennylane/pull/3454)
+  
+* Added `validate_subspace` static method to `qml.Operator` to check the validity of the subspace of certain
+  qutrit operations.
+  [(#3340)](https://github.com/PennyLaneAI/pennylane/pull/3340)
+
+* `qml.equal` now supports operators created via `qml.s_prod`, `qml.pow`, `qml.exp`, and `qml.adjoint`.
   [(#3471)](https://github.com/PennyLaneAI/pennylane/pull/3471)
 
-* Updated `zyz_decomposition` function such that it now supports broadcast operators. This
-  means that single-qubit `QubitUnitary` operators, instantiated from a batch of unitaries,
-  can now be decomposed.
+* Devices can now disregard observable grouping indices in Hamiltonians through the optional `use_grouping` attribute.
+  [(#3456)](https://github.com/PennyLaneAI/pennylane/pull/3456)
+
+* Add the optional argument `lazy=True` to functions `qml.s_prod`, `qml.prod` and `qml.op_sum` to allow simplification.
+  [(#3483)](https://github.com/PennyLaneAI/pennylane/pull/3483)
+
+* Updated the `qml.transforms.zyz_decomposition` function such that it now supports broadcast operators. This means that single-qubit `qml.QubitUnitary` operators, instantiated from a batch of unitaries, can now be decomposed.
   [(#3477)](https://github.com/PennyLaneAI/pennylane/pull/3477)
 
-* Update `OperationRecorder` to inherit from `AnnotatedQueue` and `QuantumScript` instead of `QuantumTape`.
+* Reduce usage of `MeasurementProcess.return_type`. Use `isinstance` checks instead.
+  [(#3399)](https://github.com/PennyLaneAI/pennylane/pull/3399)
+
+* The performance of executing circuits under the `jax.vmap` transformation has been improved by being able to leverage the batch-execution capabilities of some devices.
+  [(#3452)](https://github.com/PennyLaneAI/pennylane/pull/3452)
+
+* The tolerance for converting openfermion Hamiltonian complex coefficients to real ones has been modified to prevent conversion errors.
+  [(#3363)](https://github.com/PennyLaneAI/pennylane/pull/3363)
```

Comment:
I think it's the wrong PR?
```suggestion
* The tolerance for converting openfermion Hamiltonian complex coefficients to real ones has been modified to prevent conversion errors.
  [(#3367)](https://github.com/PennyLaneAI/pennylane/pull/3367)
```

Reply:
Yea that's probably what happened